src/js/owl.video.js

```diff
@@ -102,19 +102,22 @@
 	 * @param {jQuery} item - The item containing the video.
 	 */
 	Video.prototype.fetch = function(target, item) {
-		var type = target.attr('data-vimeo-id') ? 'vimeo' : 'youtube',
-			id = target.attr('data-vimeo-id') || target.attr('data-youtube-id'),
+
+		var type = target.attr('data-vimeo-id') ? 'vimeo' : target.attr('data-vzaar-id') ? 'vzaar' : 'youtube',
```

Comment:
Now that there is a nested ternary it's pretty hard to read. Could you break this into something a little more readable like:
```
var type = (function() {
				if (target.attr('data-vimeo-id')) {
					return 'vimeo';
				}
				else if (target.attr('data-vzaar-id')) {
					return 'vzaar';
				}
				else {
					return 'youtube';
				}
			})();
```
Thanks!